### ISSUE TITLE
Unable to run migration of one column is defined as ENUM (MySQL)

### ISSUE THREAD

**Issue by tdpsk** [sdispater/orator#87]
Hi,

one of the columns in my database is defined as an ENUM and was created via a migration. Now if I try to run any other migration on the same table, I get the following error:

```
  [KeyError]  
  'enum'      
              

Exception trace:
 /usr/local/lib/python3.5/site-packages/cleo/application.py in run() at line 88
   status_code = self.do_run(input_, output_)
 /usr/local/lib/python3.5/site-packages/cleo/application.py in do_run() at line 147
   status_code = command.run(input_, output_)
 /usr/local/lib/python3.5/site-packages/cleo/commands/command.py in run() at line 90
   return super(Command, self).run(i, o)
 /usr/local/lib/python3.5/site-packages/cleo/commands/base_command.py in run() at line 148
   status_code = self.execute(input_, output_)
 /usr/local/lib/python3.5/site-packages/orator/commands/command.py in execute() at line 39
   return self.handle()
 /usr/local/lib/python3.5/site-packages/orator/commands/migrations/migrate_command.py in handle() at line 42
   migrator.run(path, pretend)
 /usr/local/lib/python3.5/site-packages/orator/migrations/migrator.py in run() at line 50
   self.run_migration_list(path, migrations, pretend)
 /usr/local/lib/python3.5/site-packages/orator/migrations/migrator.py in run_migration_list() at line 68
   self._run_up(path, f, batch, pretend)
 /usr/local/lib/python3.5/site-packages/orator/migrations/migrator.py in _run_up() at line 87
   migration.up()
 /pipeline/source/migrations/2016_06_25_151138_rename_parent_column.py in up() at line 13
   table.foreign('parent_id').references('id').on('stores')
 /usr/local/lib/python3.5/contextlib.py in __exit__() at line 66
   next(self.gen)
 /usr/local/lib/python3.5/site-packages/orator/schema/builder.py in table() at line 77
   self._build(blueprint)
 /usr/local/lib/python3.5/site-packages/orator/schema/builder.py in _build() at line 147
   blueprint.build(self._connection, self._grammar)
 /usr/local/lib/python3.5/site-packages/orator/schema/blueprint.py in build() at line 30
   for statement in self.to_sql(connection, grammar):
 /usr/local/lib/python3.5/site-packages/orator/schema/blueprint.py in to_sql() at line 53
   sql = getattr(grammar, method)(self, command, connection)
 /usr/local/lib/python3.5/site-packages/orator/schema/grammars/grammar.py in compile_rename_column() at line 38
   column = connection.get_column(table, command.from_)
 /usr/local/lib/python3.5/site-packages/orator/connections/connection.py in get_column() at line 481
   return schema.list_table_details(table).get_column(column)
 /usr/local/lib/python3.5/site-packages/orator/dbal/schema_manager.py in list_table_details() at line 50
   columns = self.list_table_columns(table_name)
 /usr/local/lib/python3.5/site-packages/orator/dbal/schema_manager.py in list_table_columns() at line 33
   return self._get_portable_table_columns_list(table, table_columns)
 /usr/local/lib/python3.5/site-packages/orator/dbal/schema_manager.py in _get_portable_table_columns_list() at line 66
   column = self._get_portable_table_column_definition(table_column)
 /usr/local/lib/python3.5/site-packages/orator/dbal/mysql_schema_manager.py in _get_portable_table_column_definition() at line 35
   type = self._platform.get_type_mapping(db_type)
 /usr/local/lib/python3.5/site-packages/orator/dbal/platforms/platform.py in get_type_mapping() at line 359
   return self.INTERNAL_TYPE_MAPPING[db_type]

migrate [-d|--database DATABASE] [-p|--path PATH] [-s|--seed] [--seed-path SEED-PATH] [-P|--pretend]
```

**Comment by sdispater** [comment]
It's not specified in the documentation but modifying a column that has the `enum` type is not currently supported.

This is a limitation of the underlying database layer.

**Comment by tdpsk** [comment]
Thanks for the update. However the migration also fails if I try to modify any other column of a table that as an enum-table.

**Comment by sdispater** [comment]
OK. I will look into it and get back to you.

**Comment by sdispater** [comment]
This has been fixed in commit e2ee60012c9fe58962248180b0e530203c1c130b on the `develop` branch.
